packages/react-ui-ag/src/Listings/DesktopListing.js

```diff
@@ -40,7 +40,7 @@ export default class DesktopListing extends PureComponent {
           href={listing.url}
           className={theme.PropertyLink}
           target="_blank"
-          rel="noopener"
+          rel="noopener noreferrer"
```

Comment:
Just doing the noreferrer thing here... I think it's fine, but probably could have had a different pr for noreferrer stuff?